addons/payment/models/res_company.py

```diff
@@ -43,9 +43,9 @@ def _run_payment_onboarding_step(self, menu_id):
         # Configure Stripe
         stripe_provider = new_env.ref('payment.payment_provider_stripe')
         stripe_provider.journal_id = stripe_provider.journal_id or default_journal
-        if stripe_provider.state == 'disabled':  # The onboarding step has never been run
+        paypal_provider = new_env.ref('payment.payment_provider_paypal', raise_if_not_found=False)
+        if stripe_provider.state == 'disabled' and paypal_provider:  # The onboarding step has never been run
             # Configure PayPal
-            paypal_provider = new_env.ref('payment.payment_provider_paypal')
```

Comment:
Please copy-paste this to replace lines 44-48 :point_down: 
```py
        stripe_provider = new_env.ref('payment.payment_provider_stripe')
        stripe_provider.journal_id = stripe_provider.journal_id or default_journal
        if stripe_provider.state == 'disabled':  # The onboarding step has never been run
            # Configure PayPal
            paypal_provider = new_env.ref(
                'payment.payment_provider_paypal', raise_if_not_found=False
            )
            if paypal_provider:
                [...]
```

Reply:
@AntoineVDV updated the pr as per the suggestion